introduction_to_amazon_algorithms/xgboost_mnist/xgboost_mnist.ipynb

```diff
@@ -14,16 +14,17 @@
     "\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.    info = sm.describe_training_job(TrainingJobName=distributed_job_name)
</pre><p>Why here you add this option as a comment, and not as a flag, as you did in other notebooks (e.g. deploy_amt_model) ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3658/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
Good catch. I corrected that to stay consistent with the rest.